### ISSUE TITLE
Odd layout of objects in Nautilus title bar when window is maximized

### ISSUE THREAD

**Issue by brs17** [pop-os/gtk-theme#284]
**NOTE: Before reporting a bug, please check whether it also occurs with other themes.**
> If the bug also occurs with other themes (especially upstream default theme),
it is very likely an upstream issue.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
**Distribution (run `cat /etc/os-release`):**
Pop!_OS 19.04 (iso 14)

**Gtk+ 3.0 version - (run ```apt policy libgtk-3-0```):**
3.24.7-1ubuntu1

**Desktop environment:**
GNOME

**Pop version (run ```apt policy pop-gtk-theme```):**
4.1.3~...~dac4908

**Related Application and/or Shell theming (or name of widget)**
Nautilus

**Actual issue**
When Nautilus is full-screened, the Home and Search icon in the title bar move towards one another with blank space in the middle. In Adwaita this space shows the search bar/location text field.

Pop!_OS Screenshot:
![Screenshot from 2019-04-01 17-10-27](https://user-images.githubusercontent.com/7460774/55366161-c059ef80-54a4-11e9-9c55-b16f08091b1d.png)

**Behavior in Adwaita**

![2](https://user-images.githubusercontent.com/7460774/55366145-ae784c80-54a4-11e9-83a9-7ab428eaf632.png)

**Comment by isantop** [comment]
This is going to be a doozy. I really dislike what Adwaita does here because that "entry" is noninteractive, so they're creating a visual distinction without bringing in functionality. That said, I do agree that it looks somewhat strange when the window gets that large.

**Comment by mmstick** [comment]
That's interesting. I wonder how they came to this UX conclusion. if we need to, I can patch it. Otherwise, it may be best to match the upstream behavior.

**Comment by brs17** [comment]
I would be curious as to the discussion had for this, but in the meantime I believe it is best to match upstream behavior.

**Comment by isantop** [comment]
@jackpot51 Setting the toolbar box to expand gives us this:

![Screenshot from 2019-04-02 10-15-21](https://user-images.githubusercontent.com/5883565/55418645-4ff8b000-5530-11e9-8376-996441f8d694.png)

Basically, we get 18.10 looks and the 19.04 functionality. I think this is vastly preferable from a UX perspective than making a visually distinct, non-interactive, un-clickable area.

**Comment by isantop** [comment]
The otherwise-proposed visual changes which match upstream behavior, for reference:

![Screenshot from 2019-04-02 10-21-20](https://user-images.githubusercontent.com/5883565/55419014-212f0980-5531-11e9-987c-1f46865db203.png)

**Comment by mmstick** [comment]
I do like the UX of the second, but it would be strange to have that without it being a legitimate clickable widget.

**Comment by isantop** [comment]
Yeah, that's what ruins the UX really. But we don't really have a style for something that is there to take up space and not be interacted with. That's the definition of negative space.
